examples/hello_world/module/cpp/hello_world.cc

```diff
@@ -37,29 +37,39 @@ enum OakStatus {
   ERR_INTERNAL = 7,
 };
 
-enum ChannelHandle {
-  CHANNEL_HANDLE_UNSPECIFIED = 0,
-  LOGGING = 1,
-  GRPC_IN = 2,
-  GRPC_OUT = 3,
-};
-
 }  // namespace oak
 
 WASM_IMPORT("oak") int wait_on_channels(uint8_t* buff, int32_t count);
 WASM_IMPORT("oak")
 int channel_read(uint64_t handle, uint8_t* buff, size_t usize, uint32_t* actual_size);
 WASM_IMPORT("oak") int channel_write(uint64_t handle, uint8_t* buff, size_t usize);
+WASM_IMPORT("oak") uint64_t channel_find(uint8_t* buff, size_t usize);
 
 WASM_EXPORT int32_t oak_main() {
-  uint8_t handle_space[9] = {oak::ChannelHandle::GRPC_IN, 0, 0, 0, 0, 0, 0, 0, 0};
+  char grpc_in_name[] = "grpc_in";
```

Comment:
Why not `std::string`?

Reply:
The Wasm C++ build isn't set up to include libstdc++ stuff yet AFACT.